spec/policy/rewrite_url_captures/rewrite_url_captures_spec.lua

```diff
@@ -33,7 +41,44 @@ describe('Capture args policy', function()
           non_matching_transformation,
           matching_transformation
         }
+        local rewrite_url_captures = RewriteUrlCapturesPolicy.new(
+          { transformations = transformations }
+        )
+
+        rewrite_url_captures:rewrite()
+
+        assert.stub(set_uri_stub).was_called_with('/def')
+        assert.stub(query_params.set).was_called_with(
+          query_params, 'my_arg', 'abc')
+      end)
+    end)
+
+    describe('when there is a match but method does not match', function()
+      it('does not modifiy the path nor the params', function()
```

Comment:
```suggestion
      it('does not modify either the path or the params', function()
```

Both parts are not modified (neither of them). Is this what you mean?